File: javamodule/src/week6/section1/privateVisibility/common/PrivateMemberTest.java
package week6.section1.privateVisibility.common;

class PrivateMember {
    private int i = 1;

    private void privateFunc() {
        i += 1; // 접근 허용
    }

    void access() {
        privateFunc(); // 접근 허용
    }
}

public class PrivateMemberTest {
    public static void main(String[] args) {
        PrivateMember pc = new PrivateMember();
        //pc.i; // 접근 불가
        //pc.privateFunc(); // 접근 불가
    }
}

File: src/main/kotlin/week6/section1/privateVisibility/common/PrivateMemberTest.kt
package week6.section1.privateVisibility.common

class PrivateMember {
    private var i = 1
    private fun privateFunc() {
        i += 1         // 접근 허용
    }

    fun access() {
        privateFunc() // 접근 허용
    }
}


fun main() {
    val pm = PrivateMember() // 생성 가능
    // pm.i // 접근 불가
    // pm.privateFunc() // 접근 불가
}
